### PR TITLE
translate fill_er_up to ruby

### PR DESCRIPTION
**Pull request by yaacov**
translate fill_er_up to ruby
- users could easily chose to delete / not-delete the current providers.
- chose other providers to pre populate their db with.
- and change tokens.

### CODE REVIEW

**Comment by yaacov** [comment]
@zeari converted to script to ruby :-) run from the manageiq dir:
`../miq-helpers/fill_er_up.rb`

**Comment by zeari** [comment]
So dont delete the old file and ill merge this

File: fill_er_up.rb [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+
+delete_providers
+create_prov('localhost')
+create_prov_with_auth('Molecule', 'oshift01.eng.lab.tlv.redhat.com', token1)
+
+# ------------------------------
+# do extra stuff on the database
+# ------------------------------
+
+ContainerImageRegistry.create(:host => "exampleHost", :name => "exampleHost", :port=>"1234").save
```
Comment (moolitayer) [inline fill_er_up.rb:68]: should this be here?

File: fill_er_up.rb [hunk 2]
```diff
@@ -0,0 +1,69 @@
+#!/usr/bin/env ruby
+
+# Load Rails
+ENV['RAILS_ENV'] = ARGV[0] || 'development'
+DIR = File.dirname(__FILE__)
+require DIR + '/../manageiq/config/environment'
```
Comment (moolitayer) [inline fill_er_up.rb:6]: Seems the above assumes the script is located inside manageiq.
I think it should be based on CWD

Edit: never mind, we can always change locally

Comment (moolitayer) [inline fill_er_up.rb:6]: ~~FILE.join~~

File.join

File: fill_er_up.rb (hunk 1 above)
Comment (yaacov) [inline fill_er_up.rb:68]: > should this be here?

The line is copied from the fill_err_up.sh script, I also do need this ContainerImageRegistry in my setup.
I assume @zeari and others has a setup that use this ContainerImageRegistry ?

Comment (zeari) [inline fill_er_up.rb:68]: > The line is copied from the fill_err_up.sh script, I also do need this ContainerImageRegistry in my setup.
> I assume @zeari and others has a setup that use this ContainerImageRegistry ?

not really. Im ok with this. people usually just add\remove whatever they need.

**Pull request merged**